ingest-app/docs/api.md

```diff
@@ -220,10 +220,20 @@ Note: X-Request-Id response header always contains the same value as the CMR-Req
 
 Successful ingest responses will return an HTTP Status code of 201 for create and 200 for update/delete, and a body containing the [CMR Concept Id](#concept-id) of the item that was created, updated or deleted along with the [revision id](#revision-id).
 
-UMM-C schema validation errors are returned as warnings in the response by default. When Cmr-Validate-Umm-C request header is set to true, the ingest request will fail when there are any UMM-C validation errors.
+    {"concept-id" : "C1200000005-PROV1",
+     "revision-id" : 1,
+     "warnings" : null,
+     "existing-errors" : null}
 
-    {"concept-id":"C12345-PROV","revision-id":1,"warnings":"object has missing required properties ([\"ProcessingLevel\"])"}
+When there are existing errors allowed for progressive update, the response looks like the following:
 
+    {"concept-id" : "C1200000005-PROV1",
+     "revision-id" : 2,
+     "warnings" : [ "After translating item to UMM-C the metadata had the following issue(s): [:TilingIdentificationSystems] Tiling Identification Systems must be unique. This contains duplicates named [MODIS Tile EASE, MISR].;; [:AdditionalAttributes 2] Value [6] is not a valid value for type [DATETIME].;; [:AdditionalAttributes 3] Value [GHRSST Level 2P Global Subskin Sea Surface Temperature from the Advanced Microwave Scanning Radiometer 2 on the GCOM-W satellite] is not a valid value for type [DATETIME]." ],
```

Comment:
Warning and existing-errors are already in an array, why not use that to our advantage and keep them as items in an array?